### PR TITLE
Fixed - pagination issue

### PR DESCRIPTION
**Pull request by BhargavBhandari90**

### CODE REVIEW

File: app/assets/js/rtMedia.backbone.js [hunk 1]
```diff
@@ -392,12 +393,14 @@ jQuery( function( $ ) {
 			}
 
 			if( check_condition( 'search' ) ) {
-				var search_val = check_url( 'search' );
-				href += '?search=' + search_val;
+				if ( '' != $( '#media_search_input' ).val() ) {
```
Comment (milindmore22) [inline app/assets/js/rtMedia.backbone.js]: `!=` to `!==`

File: app/assets/js/rtMedia.backbone.js [hunk 2]
```diff
@@ -120,19 +120,20 @@ jQuery( function( $ ) {
 				if( check_condition( 'search' ) ) {
 					if ( '' != $( '#media_search_input' ).val() ) {
```
Comment (milindmore22) [inline app/assets/js/rtMedia.backbone.js]: `!=` to `!==`

**Comment by BhargavBhandari90** [comment]
@milindmore22 All changes are done. Please check.

**Comment by milindmore22** [comment]
@chandrapatel please do a final review!

**Pull request merged**